### ISSUE TITLE
Remote participation

### ISSUE THREAD

**Issue by mmccarty** [scisprints/2018_05_sklearn_skimage_dask#13]
Are there logistics in place to allow for remote participation?

**Comment by NelleV** [comment]
No, but we can attempt to set this up. Do you have any idea on how to proceed?

**Comment by mmccarty** [comment]
Yes, I thinking something like Gitter.  If there are going to be any group discussions, it would be nice to have a videocon available to follow along.

**Comment by jakirkham** [comment]
We've used [appear.in]( https://appear.in/ ) for Dask periodically. Maybe we can add room(s) for this? Google Chat's also an option. Does BIDS have some preferred software for conference calls and the like?

**Comment by jni** [comment]
:+1: for gitter and :+1: for appear.in for video. Every time I've tried appear.in it's worked flawlessly whether it was on Windows, Linux, or Mac, including screen sharing, and no login is required.

For scikit-image I will definitely be monitoring the gitter.im/scikit-image/scikit-image channel.

**Comment by hanslovsky** [comment]
I have had good experiences with both gitter and appear.in on various OSes and in different countries. Sounds good to me.

**Comment by mrocklin** [comment]
And lets use https://appear.in/dask-dev for video chat.
